src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -863,6 +863,11 @@ def load_arguments(self, _):
         c.argument('output_format', help="Set output format. Valid values are ( json | bicepparam ).")
         c.argument('include_params', help="Set include params. Valid values are ( all | required-only ).")
 
+    with self.argument_context('bicep lint') as c:
+        c.argument('file', arg_type=bicep_file_type, help="The path to the Bicep module file to lint in the file system.")
+        c.argument('no_restore', arg_type=bicep_no_restore_type, help="When set, generates the parameters file without restoring external modules.")
+        c.argument('diagnostics_format', help="Set diagnostics format. Valid values are ( default | sarif ).")
```

Comment:
If the valid values for this param are enumerable, please add `arg_type=get_enum_type` to the param, and then CLI will automatically list the available values to customers and validate the input.
```suggestion
        c.argument('diagnostics_format', arg_type=get_enum_type(['default', 'sarif']), help="Set diagnostics format.")
```